src/main/java/org/infinispan/client/rest/impl/TopologyInfo.java

```diff
@@ -0,0 +1,33 @@
+package org.infinispan.client.rest.impl;
+
+import java.util.Collection;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import org.infinispan.client.rest.configuration.ServerConfiguration;
+import org.infinispan.commons.logging.Log;
+import org.infinispan.commons.logging.LogFactory;
+
+public class TopologyInfo {
+   
+   private static final Log log = LogFactory.getLog(TopologyInfo.class);
+   
+   private AtomicInteger topologyId;
+   private Collection<ServerConfiguration> servers;
+   
+   public TopologyInfo(AtomicInteger topologyId, Collection<ServerConfiguration> servers) {

```

Comment:
Make this class immutable, turn all fields to final, use int instead of AtomicInteger